packages/web/src/components/ui/Card/Card.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import styles from './Card.module.scss';
+import clsx from 'clsx';
+import { numberWithSpaces } from '../../../utils/helpers';
+import { Bug } from 'components/icons';
+import Chip from '../Chip/Chip';
+
+export type CardProps = {
+  title: string;
+  icon?: string;
+  description?: string;
+  packageTags?: {
+    featuredPackages: string[];
+    restPackages: number;
+  };
+  featuredSites?: {
+    iconList: string[];
+    numberOfUses: number;
+  };
+  vulnerablePackage?: {
+    name: string;
+    moreCount?: number;
+  };
+  variant?: 'to-all' | 'vulnerable' | string;
+};
+
+export default function Card({
+  title,
+  icon,
+  description,
+  packageTags,
+  featuredSites,
+  vulnerablePackage,
+  variant = '',
+}: CardProps) {
+  return (
+    <div className={clsx(styles.card, styles[variant])}>
+      <div className={styles.cardTop}>
+        <header className={styles.header}>
+          {icon && (
+            <div className={styles.icon}>
+              <img src={icon} alt='' />
+            </div>
+          )}
+
+          <div className={styles.title}>{title}</div>
+        </header>
+
+        {description && <div className={styles.description}>{description}</div>}
+      </div>
+
+      {variant === 'toAll' && (
+        <button className={styles.arrowBtn} type='button'>
+          {/* TODO: properly import SVG arrow */}
+          <svg
+            width='24'
+            height='25'
+            viewBox='0 0 24 25'
+            fill='none'
+            xmlns='http://www.w3.org/2000/svg'
+          >
+            <path
+              d='M9 5.5L17 13L9 20.5'
+              stroke='#212121'
+              strokeWidth='2'
+              strokeLinecap='round'
+              strokeLinejoin='round'
+            />
+          </svg>
+        </button>
+      )}
+
+      {packageTags && (
```

Comment:
Seems like this and below would be better off with a separate react component passed in as a prop.